### PR TITLE
[docs] - Added observability platform integration options

### PR DESCRIPTION
**Pull request by james-laing**
Adding details on available observability platform integrations

### CODE REVIEW

**Comment by chrroberts-pure** [comment]
LGTM.

File: extra/o11y_platforms.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+<br>
+<img src="images/purefa_datadog_integration.png" width="50%" height="50%">
+<br>
+
+## Dynatrace
+
+https://www.dynatrace.com/hub/detail/pure-storage-flasharray
+
+Dynatrace extension 2.0 enables automates metrics ingestion and performance data corrolation. With 2x dashboards iuncluded in the bundle, users can drill down through array configurations items and monitor their fleet of arrays providing an overview of capacity utilization, data reduction and Purity version.
```
Comment (sdodsley) [inline extra/o11y_platforms.md:25]: recheck grammar, syntax and spelling

Comment (james-laing) [reply extra/o11y_platforms.md:25]: Thanks for spotting Simon - now resolved

Comment (sdodsley) [reply extra/o11y_platforms.md:25]: Not resolved

**Review by sdodsley — CHANGES_REQUESTED** [review]

File: extra/o11y_platforms.md (hunk 1 above)
Comment (james-laing) [reply extra/o11y_platforms.md]: Sorry Simon - check latest commit, fixed.

**Review by sdodsley — APPROVED** [review]

**Pull request merged**